NCRC Application and HPC End User Documentation Improvements (#22922)
idaholab/moose: modules/doc/content/ncrc/applications/hpc_application_template.md

specific application. Please refer to [inl/index.md] for information about requesting access and the
different levels available.

### (3) Viewing Results id=viewing-results

You can use HPC OnDemand to view your results remotely. Head on over to
[HPC OnDemand Dashboard](https://hpcondemand.inl.gov/pun/sys/dashboard), and select 'Interactive
Apps' dropdown menu, and then click 'Linux Desktop with Visualization'. Select your cluster (such as
Sawtooth), the amount of time you believe you need, and then click Launch. It may take some time
before your 'Visualization Job' becomes available. When it does, simply click on it, and you will be
presented within your web browser, your GUI Desktop. From here, you can open visualization
applications (such as Paraview), and open your results file.

However, in many cases it is more desirable to view the results on your local machine. This is done
by copying the results file from the remote machine to your local machine using `scp` or `rsnyc`.

!alert note title=Copying files from remote HPC machine to your machine
Copying files from an HPC cluster to your machine first requires that you follow the instructions:

> [inl/hpc_remote.md]. It is for advanced users, comfortable with modifying their SSH settings, and
> being familiar with their terminal in general.

Reviewer: ```suggestion
[inl/hpc_remote.md]. This workflow is for advanced users, comfortable with modifying their SSH settings, and
familiar with their terminal in general.
```

### (1) Create [!ac](PBS) Script

> Lets use the same example input file from above to create a simple script for scheduling. The top

Reviewer: ```suggestion
Let's use the same example input file from above to create a simple script for scheduling. The top
```
